README.md

```diff
@@ -8,9 +8,50 @@
 Adds support for fulltext search engines like Sphinx and Solr to SilverStripe CMS.
 Compatible with PHP 7.2
 
-## Maintainer Contact
-
-* Hamish Friedlander <hamish (at) silverstripe (dot) com>
+## Important note when upgrading to fulltextsearch 3.7
+
+A `canView()` check against an anonymous user (i.e. someone not logged in) and a `ShowInSearch` check is now performed
+by default against all records (DataObjects) before being added to the search index, and also before being shown in
+search results. This may mean that some records that were previously being indexed and shown in search results will
+no longer appear due to these additional checks.
+
+These additional checks have been added with data security in mind, as it's safer to assume that records failing these
+checks should not be indexed in the first place.
+
+These additional checks also mean that draft-only (unpublished) content will no longer be automatically added to the
+search index as it will now fail the `canView()` check for an anonymous user.  Previously draft-only content could be
+manually excluded from being indexed via `SearchIndex::excludeVariantState()`.  Indexed draft content was filtered from
+showing in public search results via the solr query `&fq=%2B(_versionedstage:"Live"` meaning it only requested content
+from the `Live` stage. 
+
+You can apply configuration to remove the new pre-index `canView()` check from your DataObjects if it is not necessary,
+or if it impedes expected functionality (e.g. for sites where users must authenticate to view any content.) This will
+also disable the check for descendants of the specified DataObjects. Ensure that your implementation of fulltextsearch
+is correctly performing a `canView()` check at query time before disabling the pre-index check, as this may result in
+leakage of private data.
+
+```
+SilverStripe\FullTextSearch\Search\Services\SearchableService:
+  classes_that_skip_indexing_canview_check:
+    - MyDataObject
+    - SiteTree # Will disable the check for all pagetypes
+```
+
+Skipping the pre-index `canView()` will mean that draft-only content will get added to the index.  This content
+will still get filtered out by the solr query `&fq=%2B(_versionedstage:"Live"`.  It's still recommended that you exclude
+draft content from being indexed if you don't need it via `SearchIndex::excludeVariantState()`.
+Read more on [excluding draft content](https://github.com/silverstripe/silverstripe-fulltextsearch/blob/3/docs/en/03_configuration.md).
+
+You can also use the updateIsSearchable extension point on SearchableService to modify the result of the method
+after the `ShowInSearch` and `canView()` checks have run. 
+
+It is highly recommend you run a solr_reindex on your production site after upgrading from 3.6 or earlier to purge any
+old data that should no longer be in the search index.
+
+These additional check will have an impact on the reindex performance.  It's estimated that on regular pages that
```

Comment:
```suggestion
These additional check can have an impact on the reindex performance due to additional queries for permission checks.
```

I'd leave it to the first sentence, amended above, and leave out the statement around "double time". That might be true on a stock standard recipe. I think devs can draw their own conclusions on how much impact permission checks have.

Reply:
Updated as per your recommendation